src/sentry/tasks/post_process.py

```diff
@@ -159,17 +159,22 @@ def plugin_post_process_group(plugin_slug, event, **kwargs):
     name='sentry.tasks.index_event_tags', default_retry_delay=60 * 5, max_retries=None
 )
 def index_event_tags(organization_id, project_id, event_id, tags,
-                     group_id, environment_id, **kwargs):
+                     group_id, environment_id, date_added=None, **kwargs):
     from sentry import tagstore
 
     Raven.tags_context({
         'project': project_id,
     })
 
+    create_event_tags_kwargs = {}
+    if date_added is not None:
+        create_event_tags_kwargs['date_added'] = date_added
```

Comment:
Why go through this instead of just passing it, since you handle `None` anyway?

Reply:
Good point here, this isn't really necessary based on the signature of `create_event_tags`.

I guess that I generally try to treat default argument values as an implementation detail if possible? It seems odd to me to pass `date_added=None` to `create_event_tags` to trigger the default behavior, and that also adds some sort of implicit coupling between these two functions that I'd rather avoid. (So, paranoia and pedantry, I guess?)